### PR TITLE
Fixed trailing whitespace

### PR DESCRIPTION
**Pull request by ssbarnea**
Resolve existing trailing whitespaces and prevents introduction of new ones.

### CODE REVIEW

**Review by cidrblock — APPROVED** [review]

File: licenses/LICENSE.source.json.json [hunk 1]
```diff
@@ -7,13 +7,13 @@ All rights reserved.
 MIT License
 
 Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
-documentation files (the ""Software""), to deal in the Software without restriction, including without limitation 
-the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, 
+documentation files (the ""Software""), to deal in the Software without restriction, including without limitation
```
Comment (webknjaz) [inline licenses/LICENSE.source.json.json:10]: Is it safe to edit this file? I thought it has something to do with JSON format...

**Review by webknjaz — APPROVED** [review]
This looks great, thanks! I only have one small concern that I've started a thread for. If it's not, please comment why you think so and mark it as "Resolved". Besides that, this is ready to merge.

File: licenses/LICENSE.source.json.json (hunk 1 above)
Comment (ssbarnea) [reply licenses/LICENSE.source.json.json:10]: Now that I see the double quote on that file I realise that this whitespace may be needed. Time to use git blame to find who added these and what is their purpose...

Comment (cidrblock) [reply licenses/LICENSE.source.json.json:10]: never intended to be valid json files, the extention got carried over from the source file name. 

these were renamed in #682

**Pull request merged**